<?xml version="1.0"?>
<launch>
        <arg name="input_topic" default="/Float32MultiArray_in" />
        <arg name="output_topic" default="/cloud_out" />
        <arg name="frame_id" default="/camera" />
        
        <arg name="near_clip" default= "0.001" />
        <arg name="far_clip" default= "3.5" />
        <arg name="view_angle" default= "57" />
        <arg name="height" default= "480" />
        <arg name="width" default= "640" />
        
        
	<node name="float32multiarray_to_pointcloud2" pkg="float32multiarray_to_pointcloud2" type="float32multiarray_to_pointcloud2" respawn="true" output="screen" >
	        <param name="frame_id" type="str" value="$(arg frame_id)" />
	        <param name="input_topic" type="str" value="$(arg input_topic)" />
	        <param name="output_topic" type="str" value="$(arg output_topic)" />
	        
	        <param name="near_clip" type="double" value="$(arg near_clip)" />
	        <param name="far_clip" type="double" value="$(arg far_clip)" />
	        <param name="view_angle" type="double" value="$(arg view_angle)" />
	        <param name="height" type="int" value="$(arg height)" />
	        <param name="width" type="int" value="$(arg width)" />	
	</node>
</launch>
